ddl/index.go

```diff
@@ -512,8 +515,12 @@ func (w *addIndexWorker) fetchRowColVals(txn kv.Transaction, taskRange reorgInde
 	handleOutOfRange := false
 	err := iterateSnapshotRows(w.sessCtx.GetStore(), w.table, txn.StartTS(), taskRange.startHandle,
 		func(handle int64, recordKey kv.Key, rawRow []byte) (bool, error) {
-			// Don't create endHandle index.
-			handleOutOfRange = handle >= taskRange.endHandle
+			if !taskRange.endIncluded {
+				handleOutOfRange = handle >= taskRange.endHandle
+			} else {
+				handleOutOfRange = handle > taskRange.endHandle
+			}
```

Comment:
Could we use `handleOutOfRange = handle > taskRange.endHandle` instead of line518-522?

Reply:
@zimulala No, line 519 has different condition with line 521.

Reply:
@shenli But I think line 535 done the same thing as line 519.

Reply:
@zimulala You can run all the tests because we will never add the same index twice.